components/nimbus/src/stateful/nimbus_client.rs

```diff
@@ -432,10 +432,8 @@ impl NimbusClient {
     fn get_installation_date(&self, db: &Database, writer: &mut Writer) -> Result<DateTime<Utc>> {
         // we first check our context
         if let Some(context_installation_date) = self.app_context.installation_date {
-            let res = DateTime::<Utc>::from_naive_utc_and_offset(
-                NaiveDateTime::from_timestamp_opt(context_installation_date / 1_000, 0).unwrap(),
-                Utc,
-            );
+            let res =
+                DateTime::<Utc>::from_timestamp(context_installation_date / 1_000, 0).unwrap();
```

Comment:
This seems right to me, but can someone who has used chrono more double check these changes?

Reply:
Nevermind about this one, I undid some of the changes because they required updates to rustc and now this change is no longer needed.  It might be a good idea anyways, but we don't need to do it in this PR.